Modules/Core/Common/test/itkDerivativeOperatorTest.cxx

```diff
@@ -99,6 +98,8 @@ itkDerivativeOperatorTest(int, char *[])
   // Check actual coefficient values
   if (!op1.CheckCoefficients(expected1))
   {
+    std::cerr << "Test failed!" << std::endl;
+    std::cerr << "Error in coefficients" << std::endl;
```

Comment:
Just a small comment here: `std::cerr` does flush automatically with each `<<` insertion. (For details, check: https://en.cppreference.com/w/cpp/io/cerr) This means that explicit flushing of the stream by `<< std::endl` is unnecessary. So in this case, I would suggest:

    std::cerr << "Test failed!\nError in coefficients\n";

The C++ Core Guidelines also suggest to avoid `std::endl`: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rio-endl

Reply:
@N-Dekker thanks for the review. At some point we started preferring `std::endl` over `\n` a few years ago. Cannot remember the reasons, but it dates back to the `Gerrit` days, so I have been unable to find the related comments/discussion.

Also, although at times many of my own changes are not extensive throughout the whole code base, in this case I'm reluctant to change this only for this test: this looks to me like a major style change, involving the SW Guide as well, and virtually all test argument check blocks, and error blocks. Unless a sweeping change is made (and I'm well aware of the effort this involves), I'd sacrifice performance for consistency in this case.